docs/tasks/administer-cluster/coredns.md

```diff
@@ -20,15 +20,31 @@ discovery.
 
 ## Installing CoreDNS with kubeadm
 
-In Kubernetes 1.9, [CoreDNS](https://coredns.io) is available as an alpha feature and
-may be installed by setting the `CoreDNS` feature gate to `true` during `kubeadm init`:
+In Kubernetes 1.9, [CoreDNS](https://coredns.io) is available as an alpha feature, and
+in Kubernetes 1.10 it is available as a beta feature. In either case, you may install
```

Comment:
If this is beta in 1.10, should we set the default of this feature gate to be true? cc @luxas

Reply:
Question was raised during one of the Network SIG, and the decision was that CoreDNS will be default when going GA. That means for v1.11.

NOTE: I think that before releasing with CoreDNS as default, it would be good to see the impact of this change against all the e2e tests that are based on kubeadm.

Reply:
Exactly. Unlike beta features that are driven by the API and therefore only in effect if the user accesses that API, this feature replaces an existing component when enabled. So, defaulting it to true would be essentially promoting it to GA.

Reply:
This makes sense. Totally agreed. :+1: